lib/plugins/game.js

```diff
@@ -25,10 +25,11 @@ function inject (bot, options) {
   function handleRespawnPacketData (packet) {
     bot.game.levelType = packet.levelType ?? (packet.isFlat ? 'flat' : 'default')
     bot.game.hardcore = packet.isHardcore ?? Boolean(packet.gameMode & 0b100)
-    if (bot.registry.isOlderThan('1.10')) { // gamemode is used pre 1.10 and post 1.20 but in between it's gameMode
```

Comment:
This should be a feature

Reply:
Gamemode in 1.20.6 is now of type "mapper" https://prismarinejs.github.io/minecraft-data/?d=protocol&v=1.20.6#toClient_undefined. What is that? @extremeheat

Reply:
Is that like a string then?

Reply:
Mapper is a enum. The enum values can be found in the protocol files inside minecraft-data for the relevant version. 

The current Mcpc mcdata protocol doc gen doesn't support the mapper type. Can be fixed in future by swapping it with the protodef-yaml html gen that bedrock is currently using.